content/post/data-warehouse-sla-p2.md

```diff
@@ -27,47 +28,61 @@ Here are four examples of services provided with increasing specificity:
+* Ease of use / documentation: This can be a general commitment to deliver the data in a way that is simple and easy to use. However, you can also commit to producing detailed documentation (e.g., tool tips in your business intelligence tool, data dictionary)
 
 While you should strive to build a robust analytics stack, you may never achieve the same reliability as your company's production level systems. For example, it is reasonable to expect that the systems that run the website of an ecommerce company will be more robust than their data warehouse. It costs a lot more when the website is down than when the data in the data warehouse is a few hours staler than usual. You could certainly build a robust data warehouse with production system level reliability - it all comes down to whether it makes sense to do this given your limited resources.
 
-The challenge, however, changes when you have [intelligent applications]({{< relref "data-platform.md" >}}) in production that utilize your data warehouse. In this case you must define and deliver on a significantly more robust SLA.
+The challenge, however, changes when you have [intelligent applications]({{< relref "data-platform.md" >}}) in production that utilize your data warehouse (e.g., automated post purchase emails powered by your data warehouse). In this case you must define and deliver on a significantly more robust SLA. Supporting production systems doesn't just require more resources, it requires an entirely different mentality.
 
 ### Problem reporting
-Despite our best efforts, something will go wrong and the Data team might not be the first to discover it. Clear problem reporting procedures eliminates confusion and ensures the right people know about this issue (avoid situations like "but I told Steve about the issue with X KPI last week"). This can be as simple as a Google form, a data warehouse issues email alias or a slack channel. When possible, try to avoid in person communication (or at least make sure you send an email or slack to memorialize the in person discussion) - this will be helpful if you need to write a post mortem.
+Despite your best efforts, something will go wrong and the Data team might not be the first to discover it (hopefully it is something small!). Clear problem reporting procedures eliminates confusion and ensure the right people know about this issue (avoid situations like "but I told Steve about the issue with X KPI last week" or "my most important KPI looks wrong and my Data team point of contact is on vacation"). 
```

Comment:
I think this could be read two different ways, but because you have `reporting procedures eliminates...` you should probably write `and ensures` instead of `and ensure`.

Alternatively you could write `reporting procedures eliminate confusion and ensure`